### ISSUE TITLE
splitChannels option

### ISSUE THREAD

**Issue by chiarasalerno93** [dschoon/react-waves#10]
Hi @dschoon,

Good day!
Thanks you for your project 🙂 it's so useful. I have a problem with ReactWaves component and in particular with the splitChannels field: I need to update the field splitChannels according to audio type (mono or stereo).
I would like to Render with separate waveforms for the channels only if the audio is stereo. Otherwise only one waveform is rendered.

I tried to update splitChannels field when the state changes, but the component doesn't re rendered

Can you help me?

Thank you and wishing best! 🙂

**Comment by rejochandran** [comment]
Hi @chiarasalerno93 ,

Good day!

Wavesurfer automatically loads audio into a single waveform if it is mono and loads into 2 waveforms if it is stereo provided `splitChannels: true` 

Tried an example,

loaded a mono audio and got 
<img width="376" alt="Screenshot 2019-05-28 09 06 22" src="https://user-images.githubusercontent.com/4696985/58449228-40f53f00-8128-11e9-8048-240bce33cbc0.png">

loaded a stereo audio and got
<img width="414" alt="Screenshot 2019-05-28 09 10 21" src="https://user-images.githubusercontent.com/4696985/58449279-6e41ed00-8128-11e9-8db1-61ff3da45c07.png">

My config
```
<ReactWaves
	style={{ width: "100%" }}
	className={'react-waves'}
	audioFile={this.state.audio} // switched between mono and stereo
	options={{
		...
		splitChannels: true,
		...
	}}
	...
	playing={this.state.playing}
>
```

so, are you trying to dynamically switch audio after the component has rendered or is it loaded initially?

**Comment by chiarasalerno93** [comment]
Hey @rejochandran,

Thank you for your answer! 
When I tried, I download a file audio as mono so I was convinced that it was mono and when I saw that there was two waveforms I thought that something went wrong. For this reason I tried to implement a functionality that allows to dynamically switch audio after the component has rendered.
Now I have download another file that is really mono and it works!

Thank you and wishing best! 🙂
